legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -307,6 +307,13 @@ def get_single_filing(identifier: str, filing_id: int):
             return legal_api.reports.get_pdf(rv.storage, report_type)
 
         filing_json = rv.json
+        if identifier.startswith('T'):
```

Comment:
This is not correct. This is applicable for all filings (not specific to bootstrap). Please remove this `if` and add some unit test